packages/components/atoms/f-form-field/src/components/_tests/FormField.test.js

```diff
@@ -389,6 +394,94 @@ describe('FormField', () => {
                 expect(wrapper.vm.hasTrailingIcon).toEqual(false);
             });
         });
+
+        describe('isAffixedField :: ', () => {
+            describe('when `prefix` value exists ::', () => {
+                let propsData;
+                let wrapper;
+
+                beforeEach(() => {
+                    // Arrange
+                    propsData = {
+                        inputType: 'text',
+                        prefix: '£'
+                    };
+
+                    // Act
+                    wrapper = mount(FormField, { propsData });
+                });
+
+                it('should return true`', () => {
+                    // Assert
+                    expect(wrapper.vm.isAffixedField).toBe(true);
+                });
+
+                it('should display `affixedFormField`', () => {
+                    // Arrange
+                    const affixedFormField = wrapper.find('[data-test-id="formfield-affixed-input"]');
+
+                    // Assert
+                    expect(affixedFormField.exists()).toBe(true);
+                });
+            });
+
+            describe('when `suffix` value exists ::', () => {
+                let propsData;
+                let wrapper;
+
+                beforeEach(() => {
+                    // Arrange
+                    propsData = {
+                        inputType: 'text',
+                        suffix: '£'
+                    };
+
+                    // Act
+                    wrapper = mount(FormField, { propsData });
+                });
+
+                it('should return true`', () => {
+                    // Assert
+                    expect(wrapper.vm.isAffixedField).toBe(true);
+                });
+
+                it('should display `affixedFormField`', () => {
+                    // Arrange
+                    const affixedFormField = wrapper.find('[data-test-id="formfield-affixed-input"]');
+
+                    // Assert
+                    expect(affixedFormField.exists()).toBe(true);
+                });
+            });
+
+            describe('when neither `prefix` or `suffix` value exist ::', () => {
+                let propsData;
+                let wrapper;
+
+                beforeEach(() => {
+                    // Arrange
+                    propsData = {
+                        inputType: 'text'
+                    };
+
+                    // Act
+                    wrapper = mount(FormField, { propsData });
+                });
+
+                it('should return false`', () => {
+                    // Assert
+                    expect(wrapper.vm.isAffixedField).toBe(false);
+                });
+
+                it('wrapper should display `affixedFormField`', () => {
```

Comment:
Minor but could you change this to read `should display 'affixedFormField' wrapper` please.